.github/workflows/regression-report.yml

```diff
@@ -401,30 +394,6 @@ jobs:
             exit 1
           fi
 
-      - name: Set branch
```

Comment:
Are we going to have ability to checkout to master and branch (diff.groovy is doing it) 
after this removal.

Reply:
not yet , i tried doing that. but, wasn't successfull. right now, i am using the cache job to send the branch. if you see the outputs of cache job, it sends commit sha. and, i am using the branch in generate_report. hence, this step was redundant.